Patch to remove unnecessary dependency on libva for libva-x11. libva-x11 uses
none of libva's symbols.
==========================================================================
--- a/va/Makefile.am
+++ b/va/Makefile.am
@@ -43,9 +43,9 @@
 libva_x11_backenddir   = x11
 
 libva_x11_la_SOURCES = 
-libva_x11_la_LIBADD  = $(libvacorelib) x11/libva_x11.la $(LIBVA_LIBS) $(X11_LIBS) $(XEXT_LIBS) $(DRM_LIBS) $(XFIXES_LIBS)
+libva_x11_la_LIBADD  = x11/libva_x11.la $(LIBVA_LIBS) $(X11_LIBS) $(XEXT_LIBS) $(DRM_LIBS) $(XFIXES_LIBS)
 libva_x11_la_LDFLAGS = $(LDADD)
-libva_x11_la_DEPENDENCIES = $(libvacorelib) x11/libva_x11.la
+libva_x11_la_DEPENDENCIES = x11/libva_x11.la
 
 if BUILD_DUMMY_BACKEND
 libva_dummy_backend = libva-dummy.la
--- a/test/basic/Makefile.am
+++ b/test/basic/Makefile.am
@@ -25,7 +25,8 @@
 
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
 
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib)
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib)
 
 test_01_LDADD = $(TEST_LIBS)
 test_01_SOURCES = test_01.c
--- a/test/decode/Makefile.am
+++ b/test/decode/Makefile.am
@@ -24,7 +24,8 @@
 
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
 
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib)
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib)
 
 mpeg2vldemo_LDADD = $(TEST_LIBS)
 mpeg2vldemo_SOURCES = mpeg2vldemo.c
--- a/test/encode/Makefile.am
+++ b/test/encode/Makefile.am
@@ -24,7 +24,8 @@
 
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
 
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib)
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib)
 
 h264encode_LDADD = $(TEST_LIBS)
 h264encode_SOURCES = h264encode.c
--- a/test/putsurface/Makefile.am
+++ b/test/putsurface/Makefile.am
@@ -24,7 +24,9 @@
 
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
 
-TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) -lpthread
+TEST_LIBS = $(top_builddir)/va/$(libvabackendlib) \
+  $(top_builddir)/va/$(libvacorelib) \
+  -lpthread
 
 putsurface_LDADD = $(TEST_LIBS)
 putsurface_SOURCES = putsurface.c
